simplq/src/components/pages/Admin/index.jsx

```diff
@@ -94,6 +95,7 @@ export default (props) => {
   return (
     <div className={styles['admin-content']}>
       <HeaderSection />
+      <TempQueueWarning />
```

Comment:
Lets call this component "Ribbon"

Rationale: We can reuse it multiple times with different content in future.